test/libsolidity/smtCheckerTests/overflow/signed_guard_sum_overflow.sol

```diff
@@ -10,4 +10,4 @@ contract C  {
 // ----
 // Warning 3944: (78-83): CHC: Underflow (resulting value less than -2**255) happens here.
 // Warning 4984: (78-83): CHC: Overflow (resulting value larger than 2**255 - 1) happens here.
-// Info 1391: CHC: 1 verification condition(s) proved safe! Enable the model checker option "show proved safe" to see all of them.
+// Info 1391: CHC: 2 verification condition(s) proved safe! Enable the model checker option "show proved safe" to see all of them.
```

Comment:
I think that comes from a fact that if the require statement is satisfied than there is no overflow or underflow in `x + y` expression.